homework/src/index.js

```diff
@@ -31,14 +34,83 @@
   }
 
   function main(url) {
```

Comment:
It is best to make the `main()` function definition the last one in your file, so that all functions that it calls come before it. In general, it is best to ensure that a function definition comes earlier in the file that the place where is it called.